qemu/tests/cfg/chardev_tls_encryption.cfg

```diff
@@ -25,6 +25,9 @@
             s390x:
                 RHEL.9:
                     expected_msg = "Channel binding 'tls-exporter'"
```

Comment:
With this change, i think L25~L27 can be removed.  And update the expected msg for s390x too with the old version.   
And, could you please clarify, Is it depends on host version or guest version ?

Also, I would like to comment a bit with https://github.com/autotest/tp-qemu/pull/3577/files.  @fbq815 @PaulYuuu,  I guess it is aim to apply new change for new product(RHEL.9).  In this situation, add new configuration to RHEL9 only is not friend for the future - RHEL10 and so on.

Reply:
Yes, I agree with you, I  believe the message for all platforms are the same.

Reply:
@yanan-fu the change was only for RHEL9 guest,  I don't know how this will be changed on RHEL10 yet, and we'll test RHEL8 guest on RHEL9 host, that's why I modified in that why, according to this test, maybe you could help me change it to:
@nanliu-r 
s390x:
            RHEL.8:
                expected_msg = "Channel binding 'tls-unique'"

Reply:
> @yanan-fu the change was only for RHEL9 guest, I don't know how this will be changed on RHEL10 yet

Generally,  for new change, we assume it works in the future, instead of fallback to old version.  Just FYI.

Reply:
got it ,thanks, then please help me modified the s390x part in this mr follow my last comment, thanks! @nanliu-r

Reply:
@yanan-fu Hi, after discussion with bfu offline, the behavior of s390x is the same as x86, so I remove the platform limitation. Please help to review again, thanks.